dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/demo/DemoContants.java

```diff
@@ -0,0 +1,121 @@
+package org.apache.dolphinscheduler.server.demo;
+
+public class DemoContants {
+    public static final String [] SHELL_taskDefinitionJson = {"[{\"code\":",
```

Comment:
Replace with Java Object  is better, because users can update it easily, and if someone change the Jave class, such as `TaskDefinition`, then he will update this too.